pwiz_tools/Skyline/ToolsUI/ToolInstallUI.cs

```diff
@@ -109,6 +109,7 @@ public InstallZipToolHelper(Control parent, InstallProgram installProgram)
                 return OverwriteOrInParallel(_parent, toolCollectionName, toolCollectionVersion, reportList, foundVersion, newCollectionName);
             }
 
+            // ReSharper disable once MemberHidesStaticFromOuterClass

```

Comment:
This is the only one that worries me a bit. It would be good if Brendan signed off on it.

Reply:
Could we also fix this by renaming the delegate something like "InstallProgramFunc"? Is that what it is complaining about? I guess I might prefer that.

Reply:
After reviewing, the necessary churn is not worth the fix. It is not hurting anything. Let's just go with Nick's idea to disable this warning for this one case.